doc/tutorial/examples.txt

```diff
@@ -500,8 +500,8 @@ It will be used repeatedly.
   training_steps = 10000
   
   # Declare Theano symbolic variables
-  x = T.matrix("x")
-  y = T.vector("y")
+  x = T.dmatrix("x")
+  y = T.dvector("y")
```

Comment:
So then, when we move to doctests, we should drop the more complex test and only test the case with explicit double precision. I don't think it is worth updating the test for the moment.